### ISSUE TITLE
Videos made private

### ISSUE THREAD

**Issue by skifans** [frog23/TomScottMap#20]
A couple of videos linked to are no longer available - some examples:

(YouTube Video IDs in brackets)

- INTERACTIVE MASHUP: Common Meter with Helen Arney and Jay Foreman (YE6q-e4sICs)

- The A-Z of Epidemiology: germs from Anthrax to Zoonoses. A disturbing bedtime book for kids.
 (P2C4U624Y6o)

- Ridiculous Minigolf: Crazy golf holes inspired by Portal, Minecraft, the Crystal Maze, and more (o9Ki5Vpon5c)

- The Queen's Birthday Surprise (I3n-GYlTkfo)

- Various The Technical Difficulties Show (7WkZVqYOD24, ywxC9kb727E, GUmL3B7_zDA, fEzmmDL02dY, aAQLIvRWuZY, fWeUSdjvksE, bl9WBQ2Zkhk, t9oi2fx_tdA, 2dIL2jNSZ8w, 2dIL2jNSZ8w)

- DougSoc and Jay Foreman at RAG Woodstock 2007 (UXIot6I1i80)

- Be A Safe Ninja This Winter (lCMJjehu8w8)

- Soobtitlays (_eZ2EPAdzTA)

- Twitch, Meetups, and Performing for Crowds (QY7YwPQAZ1E)

- Tom Tried A Microsoft HoloLens (cNCn-yRqNQ4)

(There may be others - just happened to notice one and did a quick check)

Should these remain on the list? Could otherwise consider leaving them but with a different icon, or point to a non-youtube source (although personally I'd be surprised if any exists). The list does act as a record that these videos did used to exist even if they are no longer viewable, which would be tricky to do otherwise.

**Comment by frog23** [comment]
That is a good question that I have been avoiding for a while. I noticed that in the review interface, some of the thumbnails wouldn't load but didn't bother looking into it. I assume the videos are set to private because of some Youtube Content-ID issues. You have pretty much the entire list, I only found two more: 
JXvZyR_pUZQ	21 Questions Wrong
1N1ydPJGGj4	American Alcohol Has To Be Radioactive

With the exceptions of The Technical Difficulties Show segments none of them have coordinates so for the primary purpose the site it wouldn't hurt to remove them (if somebody really needs to dig them up, there is always the commit history). And The Technical Difficulties Shows are still available as the full episodes, just not the individual segments anymore, so the coordinates just need to be added to the other full videos. 
I will take care of this at some point in the near future. Thanks for pointing this out.
